packages/fxa-payments-server/src/store/utils.tsx

```diff
@@ -10,11 +10,29 @@ export const mapToObject = (list: Array<string>, mapFn: Function): MappedObject
   return out;
 };
 
+type ErrorResponseBody = {
+  code?: number;
```

Comment:
Thinking this might be more elegant as a couple separate types for response body - one that has all these properties and one that's more lenient for whatever comes back as an error